packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -208,21 +209,23 @@ class IndividualAction extends Component {
     (disabled, item, userData) =>
       isFunction(disabled) ? disabled(item, userData) : disabled
   )
-
   _executeAction = () => {
     const p = this.props
     return p.handler(p.item, p.userData)
   }
```

Comment:
I'm wondering whether this indirection is still relevant.

Reply:
The problem is that `ActionButton` will pass `({ item, userData })` while all the handlers expect `(item, userData)`. So we'd either need to change all the handlers or do something like:
```js
_executeAction = ({ item, userData }) => this.props.handler(item, userData)
```
which is basically equivalent and doesn't get rid of the extra indirection.

Reply:
Does that mean a lot of changes?
Because it would improve homogeneity between `handler` and `redirectOnSuccess`.

Reply:
Quite a bunch but feasible.
Another thing is that many handlers are very simple and only need the item which allowed us, for example, to pass: `handler: deleteSr` instead of `handler: ({ item }) => deleteSr(item)`.

Reply:
Makes sense, let's keep it that way then :smiley: